server/app/views/admin/programs/ProgramBaseView.java

```diff
@@ -24,8 +27,38 @@
 
 abstract class ProgramBaseView extends BaseHtmlView {
 
-  /** Renders a div with internal/admin program information. */
-  protected final DivTag renderProgramInfo(ProgramDefinition programDefinition) {
+  private static final String HEADER_BUTTON_STYLES =
```

Comment:
(non-blocking) If we plan to use this style elsewhere on admin pages, we can add it to https://github.com/civiform/civiform/blob/749f98a7af7a45f27b44c6eb7c9c1d81d5e4bb28/server/app/views/style/AdminStyles.java

Reply:
Added to AdminStyles since it seems good to keep all the styles in the same place regardless.